public/js/reducers/breakpoints.js

```diff
@@ -138,6 +134,26 @@ function update(state = State(), action: Action) {
   return state;
 }
 
+function toggleBreakpoints(state: any, action: any) {
```

Comment:
Where is the error? We really shouldn't digress on type coverage. It's already very poor.